README.md

```diff
@@ -444,6 +445,24 @@ Note: For modules use the [@import](http://clang.llvm.org/docs/Modules.html#usin
 #import "NYTUserView.h"
 ```
 
+## Protocols
+
+In a [delegate or data source protocol](https://developer.apple.com/library/ios/documentation/General/Conceptual/CocoaEncyclopedia/DelegatesandDataSources/DelegatesandDataSources.html), the first parameter to each method should be the object sending the message.
+
+This helps disambiguate in cases when an object is the delegate for multiple similarly-typed objects, and it helps clarify intent to readers of a class implementing these delegate methods.
+
+**For example:**
+
+```objc
+- (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath;
```

Comment:
Many of the examples in the guide use `NYT`-based classes for cases like this so as not to confuse people and steer people towards the fact that this guidance is to be used when implementing your own objects. I don’t know how much it matters, or how much you care, but it might be nice to make this a non-UIKit example if we want to stick with that pattern.

That was rambly, but hopefully made some sense. Let me know if it didn’t and I can make an illustrative PR.